CustomerOrganizer/CustomerOrganizer/MainWindow.py

```diff
@@ -1,7 +1,31 @@
 import tkinter
+from tkinter import *
+import FileIO
+from FileIO import *
 
-class Application(tkinter.Frame):
+class MainWindow(tkinter.Tk):
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.pack()
\ No newline at end of file
+        self.toolBar()
+
+        self.customer = FileIO.FileIO()
```

Comment:
Move this above the `self.toolBar()` line. Also getting an "'FileIO' has no attribute 'FileIO'" error. Change to `self.customer = FileIO()` to get it to work.

Reply:
yeah somethings up with VS color coding, FileIO is colored as a library and FileIO.FileIO is colored as a class. ill keep it in mind